### DISCUSSION TITLE
"Less than" symbol not being shown in Copilot Chat

### DISCUSSION THREAD

**Discussion by Unbreathable** [community/community#60431] [Copilot Conversations] [answered]
### Select Topic Area

Bug

**Edit:** As of writing this it's already fixed in the newest version of Copilot. I don't know how to close this, so if anyone who knows how to do that reads this, please close this "post".

### Body

Hello Copilot-Team,
I've recently been using Copilot Chat a ton to get stuff done and it is really useful, but when programming with rust and asking the Chatbot to "Convert a vec<f64> to a vec<f32>" the "less than" symbols in the prompt didn't show up in the interface. I'm using Visual Studio Code.
![grafik](https://github.com/community/community/assets/70802809/54d8a6fd-aefe-41a4-b47a-59899b1ac108)
Copilot Chat still seems to recognize them as characters, but it just becomes a little confusing if you have a longer conversation with it. This isn't a high priority bug, but I think it could be a quick fix and would certainly help clear up confusion in case someone else also found it.

**Comment by Akash1134** [comment, marked as answer]
Hi @Unbreathable ,

Thank you for reporting this issue. Though, Copilot Chat is still in its public preview and it might not work perfectly in some cases. I have forwarded this to our engineers to look into.
